Worker/src/Common/WorkerHelpers.cs

```diff
@@ -32,19 +32,3 @@ public static ILoggerFactory GetDefaultLoggerFactory(IConfiguration configuratio
     return LoggerFactory.Create(loggingBuilder => loggingBuilder.AddSerilog(loggerConfig));
```

Comment:
Please remove the getDefaultLoggerFactory method. It can enter in conflict with different version of serilog used by the end user